tests/load-tests/ci-scripts/collect-results.sh

```diff
@@ -121,28 +120,36 @@ python3 -m pip install -U pip
 python3 -m pip install -e "git+https://github.com/redhat-performance/opl.git#egg=opl-rhcloud-perf-team-core&subdirectory=core"
 
 echo "Collecting monitoring data..."
-mstart=$(date --utc --date "$(status_data.py --status-data-file "$monitoring_collection_data" --get timestamp)" --iso-8601=seconds)
-mend=$(date --utc --date "$(status_data.py --status-data-file "$monitoring_collection_data" --get endTimestamp)" --iso-8601=seconds)
-mhost=$(oc -n openshift-monitoring get route -l app.kubernetes.io/name=thanos-query -o json | jq --raw-output '.items[0].spec.host')
-status_data.py \
-    --status-data-file "$monitoring_collection_data" \
-    --additional ./tests/load-tests/cluster_read_config.yaml \
-    --monitoring-start "$mstart" \
-    --monitoring-end "$mend" \
-    --prometheus-host "https://$mhost" \
-    --prometheus-port 443 \
-    --prometheus-token "$(oc whoami -t)" \
-    -d &>$monitoring_collection_log
-set +u
-deactivate
-set -u
+if [ -f "$monitoring_collection_data" ]; then
```

Comment:
FYI If the file does not exist, it will create it (if it's directory exists).

Reply:
Yes, but if I understand it correctly, if the `load-test.json` does not exist, it would not be able to get the `mstart` a `mend` values from it that are later used to set `monitoring-start` and `monitoring-end` and so we wouldn't  be able to get the monitoring data from the test's time window, anyway. Please correct me if I'm wrong.

Reply:
Aaah, you are right!